### ISSUE TITLE
Strange regexp for step openPage

### ISSUE THREAD

**Issue by PavelAlennikov** [sbtqa/page-factory#109]
ru.sbtqa.tag.pagefactory.openPage=^(?:пользователь |он |)(?:находится на странице|открывается страница|открывается вкладка мастера) \"([^\"]*)\"$

This regexp matches to strange step like this: пользователь|он открывается страница ...

**Comment by kosteman** [comment]
Yes, you're right. But you can spell a "пользователь находится на странице "([^"]*)"" step.
You can make it more accurate by pull request.
